addons/mail/static/src/components/follower/follower.xml

```diff
@@ -3,16 +3,16 @@
 
     <t t-name="mail.Follower" owl="1">
         <t t-if="follower">
-            <div class="o_Follower" t-attf-class="{{ className }}" t-on-click="props.onClick" t-ref="root">
-                <a class="o_Follower_details d-flex" t-att-class="{ 'o-inactive': !follower.isActive }" href="#" t-on-click="_onClickDetails">
-                    <img class="o_Follower_avatar" t-att-src="follower.partner.avatarUrl" alt="Avatar"/>
+            <div class="o_Follower justify-content-between d-flex p-0" t-attf-class="{{ className }}" t-on-click="props.onClick" t-ref="root">
```

Comment:
`d-flex` should be before `justify-content-between`

Reply:
According to the SCSS guidelines, flex properties should not be located before the `display` property ?
https://github.com/odoo/odoo/wiki/SCSS-coding-guidelines

Reply:
Debatable, guidelines are not very clear about it... `display` should be placed  after position rules, it's true, but "flex rules" should be placed before.
I consider `display: flex` a "flex rule", moreover classes like `justify-content-between` makes sense if `d-flex` is applied only.

If, instead, the element was `align-self-baseline d-block`  I would have agreed.

Reply:
Ok, I updated the branch 👍